proto/push/service.proto

```diff
@@ -95,9 +97,139 @@ message GetPushResponse {
 }
 
 service PushService {
-  rpc ListPushes(ListPushesRequest) returns (ListPushesResponse) {}
-  rpc CreatePush(CreatePushRequest) returns (CreatePushResponse) {}
+  rpc ListPushes(ListPushesRequest) returns (ListPushesResponse) {
+    option (google.api.http) = {
```

Comment:
Can you check why this file has not been updated?
https://github.com/bucketeer-io/bucketeer/blob/main/api-description/web-api.swagger.yaml

Reply:
Oops, missing declaration in this line: https://github.com/bucketeer-io/bucketeer/blob/4c87ff6cfc82fe0e1e7098cbf9081ce10dab1d0a/proto/Makefile#L59

I have regenerated swagger!